missile.py

```diff
@@ -1,19 +1,48 @@
+import pygame
+import math
```

Comment:
These two libraries are already imported in by the constants.py file.

Reply:
Oh whoops, lets delete these then